packages/@aws-cdk/aws-codestarnotifications/lib/source.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * The source type of the notification rule.
+ */
+export enum SourceType {
```

Comment:
The type of the source should be a string. Let's remove this enum completely.

Reply:
Removed.